src/components/shared/Validations.tsx

```diff
@@ -73,3 +77,15 @@ export const Validations: React.FunctionComponent<IValidations> = ({
     </>
   );
 };
+
+export const Format: React.FunctionComponent<{ schemaNode: JSONSchema4 }> = ({ schemaNode }) => {
+  return (
+    <div
+      {...(typeof schemaNode.type === 'string'
+        ? { className: 'ml-2 ' + PropertyTypeColors[schemaNode.type as string] }
```

Comment:
```suggestion
        ? { className: `ml-2 ${PropertyTypeColors[schemaNode.type as string]}` }
```
one could also use classnames, IIRC it's already used in a few spots